src/memoizeWith.js

```diff
@@ -0,0 +1,43 @@
+var _arity = require('./internal/_arity');
+var _curry2 = require('./internal/_curry2');
+var _has = require('./internal/_has');
+
+
+/**
+ * A customisable version of R.memoize: takes an additional function that will
```

Comment:
I purposely chose the colon so I wouldn't use the word 'that' twice in such close proximity.